src/Components/DrillPage.js

```diff
@@ -1,25 +1,32 @@
 import React, { Component } from 'react';
-import { ScrollView, StyleSheet, View, Text, Button, ImageBackground, TouchableOpacity } from 'react-native';
+import { ScrollView, StyleSheet, View, Text, Image, ImageBackground, TouchableOpacity } from 'react-native';
 import { connect } from 'react-redux';
 
 class DrillPage extends React.Component {
+  _toggleFavorite() {
```

Comment:
By the way it looks like this function is not used, you may remove it and use only `this.props.toggleFavorite(drill)`